exercise_0/main.cpp

```diff
@@ -59,6 +59,7 @@ int     read_from_file(char *file_name) {
         return 1;
     }
     words_checker(input_file, nullptr);
+    input_file.close();
```

Comment:
Not needed. Please see "RAII" patern

Reply:
I cannot add comment directly to line 56.
Please open file in constructor - again, RAII pattern.

Reply:
Fix is present in **9ed9ba**2500aa69d54bf65f5b83708fc3966015ff commit.